modin/core/io/io.py

```diff
@@ -436,6 +438,8 @@ def read_sql(
         parse_dates=None,
         columns=None,
         chunksize=None,
+        dtype_backend=no_default,
+        dtype=None,
```

Comment:
Why are the arguments not propagated into the original method?

Reply:
Fixed